src/wrapper.cpp

```diff
@@ -60,7 +63,14 @@ Rcpp::DataFrame RamerDouglasPeucker(Rcpp::NumericVector x, Rcpp::NumericVector y
         std::size_t index = indicesToKeep[i];
         xOut[i] = x[index];
         yOut[i] = y[index];
+        indicesToKeep[i] += 1; // index from 1 before returning to R
```

Comment:
I prefer the comment on a separate line above

Reply:
Thanks, I've applied this change.